Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/2_1_2.md

```diff
@@ -1,6 +1,38 @@
-<~PLATFORM>
+#### Breaking Changes
+
+##### Group-IB Threat Intelligence
+
+- Removed the **exclude_combolist** parameter (previously named "Exclude All with Combolist type"). This parameter has been replaced with the new **Include combolist type in data** parameter.
```

Comment:
The release note entry `- Removed the **exclude_combolist** parameter...` uses a non-standard prefix. Please rephrase it to use an approved template, such as `Deprecated the `.